spec/rails_helper.rb

```diff
@@ -60,3 +60,10 @@ def sign_in(member)
   fill_in 'Password', with: member.password
   click_button 'Log in'
 end
+
+def create_vendor(vendor)
+  visit vendors_path
+  fill_in 'Vendor Name', with: vendor.vendor_name
+  select('Breakfast', from: 'vendor_category_ids')
+  click_button 'Add Vendor'
+end
```

Comment:
Hey! We're not exactly sure how you would like us to modify this. The create_vendor(vendor) method has the vendor param being passed in from a let statement with a factory located on the test page. We also have a category factory there, but we didn't use it in the method definition because we found that it broke when we tried to use it...both in database population and testing.